aea/connections/base.py

```diff
@@ -277,12 +277,12 @@ def from_config(
         """
         configuration = cast(ConnectionConfig, configuration)
         directory = cast(Path, configuration.directory)
-        load_aea_package(configuration)
         connection_module_path = directory / "connection.py"
         if not (connection_module_path.exists() and connection_module_path.is_file()):
             raise AEAComponentLoadException(
                 "Connection module '{}' not found.".format(connection_module_path)
             )
+        load_aea_package(configuration)
```

Comment:
Can you explain?

Reply:
it's done for one test. with load_aea_package update, i perform extra check for dir exists and one negative test fails

now it's ok cause connection module check done before the module loading attempt.